app/src/main/java/org/oppia/app/walkthrough/WalkthroughActivity.kt

```diff
@@ -23,4 +23,8 @@ class WalkthroughActivity : InjectableAppCompatActivity() {
     activityComponent.inject(this)
     walkthroughActivityPresenter.handleOnCreate()
   }
+
+  override fun pageNumber(pageNo: Int) {
```

Comment:
Rename `pageNumber` to `changeTo` and inside this you should pass the enum that you created earlier. eventually it will be an int value only but 0,1,2 does not given any information but instead enam gives you precise information.

Reply:
done